source/index.html

```diff
@@ -1,12 +1,172 @@
 <!DOCTYPE html>
-<html lang="ru">
-  <head>
-    <meta charset="utf-8">
-    <title>HTML Academy: Кэт энерджи</title>
-  </head>
-  <body>
+<html class="page" lang="ru">
 
-    <p>Репозиторий создан для обучения на профессиональном онлайн‑курсе «<a href="https://htmlacademy.ru/intensive/adaptive">HTML и CSS. Адаптивная вёрстка и автоматизация</a>».</p>
+<head>
+  <meta charset="utf-8">
+  <title>Интернет-магазин питания для котов "Кэт энерджи"</title>
+  <link rel="stylesheet" href="">
+  <link rel="stylesheet" href="">
+  <link rel="icon" href="">
+  <link rel="icon" href="" type="image/svg+xml">
+  <link rel="apple-touch-icon" href="">
+</head>
+
+<body class="page__body">
+  <header class="page-header">
+    <a class="page-header__logo-icon" href="#"><img class="page-header__logo-image" src="img/logo_icon.svg" width="34"
+        height="38" alt="Иконка-логотип магазина Кэт энерджи">
+    </a>
+    <div class="main-logo">
+      <a class="main-logo__link" href="#"><img class="main-logo__logo-image" src="img/logo_main_top.svg" width="101"
+          height="18" alt="Логотип магазина Кэт энерджи">
+      </a>
+    </div>
+    <nav class="main-nav">
+      <button class="main-nav__toggle" type="button"><span class="visually-hidden">Открыть меню</span></button>
```

Comment:
Эта кнопка не нужна до полной отработки скрипта схлопывающего мобильную навигацию. Поэтому давай спрячем её недоступно атрибутом.
```suggestion
      <button class="main-nav__toggle" type="button" hidden><span class="visually-hidden">Открыть меню</span></button>
```